external-crates/move/crates/move-disassembler/src/disassembler.rs

```diff
@@ -279,13 +279,14 @@ impl<'a> Disassembler<'a> {
         bcode_map_gen: bool,
     ) -> Result<SourceMap> {
         // NB: The order in which these are called is important as each function is effectful.
-        let mut bcode_map = self.print_header(buffer)?;
+        let mut res = Some(self.print_header(buffer)?);
```

Comment:
Nit: consider if this needs to make the source map at all, but I think this is fine as-is if you'd prefer to keep it like that.

Reply:
I was thinking about it decided to let it simply return largely empty `SourceMap` rather than `None` is that on the client side we then don't have to do another check (and possibly user-facing message) when unwrapping (e.g. for printing)